### ISSUE TITLE
Quasi global coverage for "Region"

### ISSUE THREAD

**Issue by heroldn** [ACDguide/invenioClim#28]
For "Region" one can currently select different continents or other named regions. However, there are some products (e.g. from satellite) that provide coverage from say -50 to 50 degrees latitude. This is normally called something like "quasi global" coverage. 
This might not be possible but it would be nice to be able to specify a text string with the latitude/longitude coverage. e.g. instead of selecting "Global" I could type in "-50 to 50 degrees latitude".

**Comment by paolap** [comment]
Theoretically is possible to do it already as there's a specific geospatial object in the record metadata itself. However, it's not visualised in the forms and it would be tricky to add a proper query to it. The from date to date temporal range already gives you an idea that querying what falls into a specific region is not trivial. I think someone else eventually might add this feature to the main portal and we'll be able to leverage it. In the meantime we could add more region like the semi-global you pointed out.

**Comment by heroldn** [comment]
Yes a "semi global" or "quasi global" might do for now. It would cover datasets that only go to certain latitudes or as Claire pointed out satellite swaths. And I think this would help narrow down users' searches sufficiently.
